### PR TITLE
Updating the build.md with some formatting and language improvements

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: build.md [hunk 1]
```diff
@@ -54,7 +55,7 @@ Tarball content for the Linux host (using libvirt)
 Tarball content for the Mac host (using VirtualBox)
 ---------------------------------------------------
 
-- Since `test1-base` is backing file for `test1-master-0` and also for the worker.
+- `test1-base` is the backup file for master and worker disk image.
```
Comment (praveenkumar) [inline build.md]: It is a `backing` file not the `backup` check the output of the `qemu-img info` command.

File: build.md [hunk 2]
```diff
@@ -70,7 +71,7 @@ Format specific information:
     refcount bits: 16
 ```
 
-- First merge the snapshot and the backing file into standalone disk image both for master and worker.
+- First merge the snapshot and the backup file into standalone disk image both for master and worker.
```
Comment (praveenkumar) [inline build.md]: Same here.

File: build.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
-This document explains about how we are generating the disk images and also tarball creation.
+# Build Process
+### This document explains about how we are creating the disk images and the tarball
 
-- Take a tag release from Openshift installer and build it on the Linux box.
+- Take a tagged release from Openshift installer and build it on a Linux machine.
```
Comment (praveenkumar) [inline build.md]: :+1:

**Comment by LalatenduMohanty** [comment]
@praveenkumar fixed it.

**Pull request merged**